app/Http/Controllers/PostController.php

```diff
@@ -30,6 +30,13 @@ public function create()
     public function store()
     {
 
+
+      $this->validate(request(),[
+        'name' => 'required',
+        'email' => 'required',
+        'message' => 'required'
+      ]);
+
       //die and dump
        // dd(request()->all());
```

Comment:
:mask::mask::mask::mask:
Commented code in the master branch should be avoided. Regular comments like `//die and dump` is just fine, but it's just that it's commented out code that's not typically okay.